apps/web-giddh/src/app/ledger/components/import-statement/import-statement.component.ts

```
@@ -120,14 +215,70 @@ export class ImportStatementComponent implements OnDestroy {
+     * @memberof ImportStatementComponent
+     */
+    public searchAccount(query: string = '', page: number = 1): void {
+        if (page === 1) {
+            this.accountSearchResponse = [];
+        }
+        this.accountSearchRequest.q = query;
+        this.accountSearchRequest.page = page;
+        this.accountSearchRequest.isLoading = true;
+
+        let requestObject = cloneDeep(this.accountSearchRequest);
+        delete requestObject.isLoading;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)